Bukkit/src/main/java/com/plotsquared/bukkit/listener/PaperListener.java

```diff
@@ -446,4 +450,11 @@ public void onBeaconEffect(final BeaconEffectEvent event) {
         }
     }
 
+    private boolean getBooleanFlagValue(@NonNull FlagContainer container,
```

Comment:
There's no longer a need to do this as the flag is now added to the GlobalFlagContainer

Reply:
Should I revert back to the old way there or is this way also okay?